projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.ts

```diff
@@ -56,7 +59,12 @@ export class TimeSeriesChartComponent implements OnInit, AfterViewInit, OnChange
     };
   };
 
-  constructor(@Self() private element: ElementRef) {}
+  constructor(@Self() private element: ElementRef, private executionService: ExecutionsService) {}
```

Comment:
Current Angular version allows to inject dependencies directly in fields, instead of constructor
```suggestion
private _element = inject(ElementRef);
private _executionsService = inject(ExecutionsService);
```


Also prefix injected dependencies with underscore:
https://exense.atlassian.net/wiki/spaces/DEVELOPMEN/pages/13762724/Code+style#_-prefix-for-injectables